samples/hosting/docker/Core_7/Sender/Program.cs

```diff
@@ -20,6 +20,12 @@ static IHostBuilder CreateHostBuilder(string[] args)
         {
             return Host.CreateDefaultBuilder(args)
                 .UseConsoleLifetime()
+                .ConfigureLogging(logging =>
+                {
+                    logging.AddConsole();
+                    logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Information);
```

Comment:
I guess it doens't matter much but I would just add a using above and then use `LogLevel.Information`